utils/docker/Dockerfile.centos.7

```
@@ -62,7 +62,7 @@ RUN yum -y install epel-release; \
         python-pep8 python-pygit2 python2-pygithub python-requests     \
         readline-devel scons sg3_utils ShellCheck yasm pciutils        \
         valgrind-devel python36-pylint man java-1.8.0-openjdk maven    \
-        json-c-devel python36-requests
+        json-c-devel python36-requests lz4 lz4-devel
```

```suggestion
        json-c-devel python36-requests lz4-devel
```

`lz4-devel` will already requires the `lz4` library.  Letting it be implicit means we don't have to do anything if upstream changes packaging for any reason.